helm/scaphandre/templates/daemonset.yaml

```diff
@@ -36,6 +36,10 @@ spec:
             {{- end }}
             {{- end }}
         env:
+        - name: KUBERNETES_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
```

Comment:
This uses the Downward API to set the node name.

https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/